### DISCUSSION TITLE
Docs: Better explanation needed of rendering mode vs. deployment target choices

### DISCUSSION THREAD

**Discussion by mitya33** [nuxt/nuxt#9493] [General]
Hey guys,

I'm fairly new to Nuxt and have been struggling to understand precisely how rendeirng mode and deployment target differ. They seem to have some implied overlap and it took [this excellent answer](https://stackoverflow.com/a/63638062/1352288) on Stack Overflow for me to finally start to understand.

I feel the Nuxt docs really don't cover these options well at all. (For example, from [this page](https://github.com/nuxt/create-nuxt-app), which lists setup options such as these two, the links to Universal and SPA seem to go to wrong URLs; the page exists, but the hash link doesn't jump down to anything on the page.)

I'd love for the docs to discuss the four permutations these two options offer so it's clear precisely what you're choosing and how they differ.

**Comment by ryan-jagger** [comment]
Would love some official clarification on this as well. I think a flow-chart like graphic would be super helpful here.

**Comment by kissu** [comment]
There is maybe not enough visibility on this video: https://youtu.be/YdELCvnBQ4E
Or maybe it is a bit too long and you'd wish a shorter, visual explanation of it? Sure thing!

Ofc, Atinux also did a wonderful visual presentation here (a bit old but still relevant): https://youtu.be/OF3A_arLJ8k?t=60
This is how I've actually learned a lot of what is happening.

This one is more up to date and also talks about `target: static`: https://youtu.be/9CkfX6LgNUU
Lastly, this talk is interesting too: https://www.vuemastery.com/conferences/vueconf-us-2020/static-is-the-new-dyncamic

It can sometimes be a bit hard to understand but it's also because there is plenty of options.
To this, can also be added some serverless rendering (soon).

PS: glad to see you really interested into the JS/Nuxt ecosystem @mitya33! 💪🏼
